mne_bids/tests/test_path.py

```diff
@@ -226,7 +226,7 @@ def test_get_entities_from_fname(fname):
 
 @pytest.mark.parametrize('fname', [
     'sub-01_ses-02_task-test_run-3_split-01_meg.fif',
-    ('/bids_root/sub-01/ses-02/meg/' +
+    ('/bids_root/sub-01/ses-02/meg/',
```

Comment:
Now with the comma this makes it a tuple! I guess you don't want the comma there! Just remove it, and Python will assume the string continues in the next line.